tests/test.cc

```diff
@@ -1,2 +1,30 @@
+#include "node.h"
+
 #define CATCH_CONFIG_MAIN
 #include "catch.hpp"
+
+TEST_CASE( "Node is checked", "[Node]" ) {
+	Node test (123, {11.2, 22.3, 33.4});
```

Comment:
use meaningful variable names, `test` is not meaningful. What's test? use `node` instead. I would suggest using a `unique_ptr` to create a Node object. Set id as a variable and the coordinates as variables.